install.cmd

```diff
@@ -0,0 +1,42 @@
+@echo off
+REM Copyright (c) 2012-2017, EnterpriseDB Corporation.  All rights reserved 
+REM PostgreSQL server psql runner script for Windows
+REM Northwind Windows PostgreSQL database install
+
```

Comment:
The northwind database is from microsoft. 
It would be best to remove the entrepriseDB copyright in the script, so as to avoid any confusion.